plugins/labormanager/labormanager.cpp

```diff
@@ -2281,6 +2307,26 @@ command_result labormanager(color_ostream &out, std::vector <std::string> & para
         print_labor(labor, out);
         return CR_OK;
     }
+    else if (parameters.size() == 2 && parameters[0] == "ignoremarker")
+    {
+        if (!enable_labormanager)
+        {
+            out << "Error: The plugin is not enabled." << endl;
+            return CR_FAILURE;
+        }
+
+        df::unit_labor labor = lookup_labor_by_name(parameters[1]);
+
+        if (labor == df::unit_labor::NONE)
```

Comment:
You're also using NONE to mean "not set", so this makes it impossible to actually unset the marker. Can you change that? A special case for non-case-sensitive "NONE" might be enough... not sure if the underlying data that `lookup_labor_by_name` uses makes it easy to check for invalid names.